# Message that carries accelerometer and gyroscope information

float32[] accel_data
float32[] gyro_data
